traffic_ops/install/bin/_postinstall

```diff
@@ -220,6 +220,11 @@ sub generateCdnConf {
     if (exists $cdnConfiguration{base_url}) {
         $cdnConf->{to}{base_url} = $cdnConfiguration{base_url};
     }
+    if (exists $cdnConfiguration{port}) {
+        $cdnConf->{"traffic_ops_golang"}->{port} = $cdnConfiguration{port};
```

Comment:
not a biggie since it's scattered thru all our perl code:   the arrow between braces is not necessary..  this is cleaner:
```
$cdnConf->{traffic_ops_golang}{port} = $cdnConfiguration{port};
```

Reply:
Fixed.